### DISCUSSION TITLE
How to uninstall?

### DISCUSSION THREAD

**Discussion by eliich** [oobabooga/text-generation-webui#5315] [Q&A] [answered]
How to uninstall? can I simply just delete the text-generation-webui-main or do I need to uninstall it somehow. I want to cleanly uninstall it.

**Comment by ResearchForumOnline** [comment]
First delete the folder, then remove any packages you do not want, deleting the main folder is good enough.

**Comment by araleza** [comment, marked as answer]
Yeah, just deleting the folder should get the job done. It uses a virtual environment, so any packages it installed are contained in a subdirectory of the main folder
